mypyc/test-data/irbuild-basic.test

```diff
@@ -1926,9 +1926,9 @@ L0:
     r5 = get_element_ptr r1 ob_item :: PyListObject
     r6 = load_mem r5 :: ptr*
     set_mem r6, r2 :: builtins.object*
-    r7 = r6 + WORD_SIZE*1
+    r7 = r6 + 8
     set_mem r7, r3 :: builtins.object*
-    r8 = r6 + WORD_SIZE*2
+    r8 = r6 + 16
```

Comment:
**Drive-by comment**: `--update-data` unfortunately doesn't know that `WORD_SIZE` is platform dependent and can't simply be substituted with the runtime size. https://github.com/mypyc/mypyc/issues/859.

You're gonna have to fix all the cases where WORD_SIZE was replaced by hand. Sorry.

Reply:
Ah that explains it, thanks.